lib/config.js

```diff
@@ -141,6 +141,11 @@ function getLocalConfig(thisConfig, directory) {
         localConfigFile = localConfigFiles[i];
         localConfig = loadConfig(localConfigFile);
 
+        // Don't consider the personal config file in the home directory.
```

Comment:
Maybe move before `localConfig =`? No sense in reading the file just to throw it away.